### ISSUE TITLE
Should keep index consistency across partitions

### ISSUE THREAD

**Issue by linhong-intel** [Intel-bigdata/OAP#445]
for example:
Create index_1 on column A in partition A
Create index_1 on column B in partition B

This is allowed by OAP, but this inconsistency will cause problem later, say, refresh index.

**Comment by linhong-intel** [comment]
@songzhan01 
you can copy you test case here for this issue.

**Comment by LuciferYang** [comment]
@linhong-intel sumbit a task to check in parallel whether all existing index_1 point to column A?

**Comment by adrian-wang** [comment]
This is result from that we allow operation on a single partition. Some other issues may be caused here, I'll create another issue tracking this, thanks!
